examples/helloworld/cwf.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
There is E2E test that deploys helloworld resources. Without Argo CRD installed it will fail. Please remove.

Reply:
sure. I will remove them.